_layouts/private.html

```diff
@@ -12,6 +12,7 @@
             {% include private-redirect.html %}
           {%- endcapture %}
           {{ private-redirect | markdownify }}
+          {%- include edit-link.html -%}
```

Comment:
```suggestion

```
Can we remove on private pages? It doesn't seem helpful there as it just takes you to the text to edit privately. It's a more labourious process to move over a note once it's published and I presume easier to do through github ?